client/settings/payment-method-settings/test/index.js

```diff
@@ -89,7 +89,7 @@ describe( 'PaymentMethodSettings', () => {
 	test( 'renders banner at the top', () => {
 		render( <PaymentMethodSettings methodId="payment_request" /> );
 
-		const banner = screen.queryByTitle( 'WooCommerce Payments' );
```

Comment:
I would add additional test that will cover a case when Jetpack IDC modal or notification is shown

Reply:
Addressed 00190b3cf0bd5179f6c3179a5b8054b1100d72c0 and with a little improvement for this test file f1af4ccab8957ce69ef744b0a6ab0cbcac794b8c